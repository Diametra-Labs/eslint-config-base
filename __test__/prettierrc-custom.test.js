const prettierConfig = require('../.prettierrc.js');

describe('The prettierrc custom file should not change', () => {
    test('It should response with custom configuration', () => {
        expect(prettierConfig).toMatchSnapshot();
    });
});
