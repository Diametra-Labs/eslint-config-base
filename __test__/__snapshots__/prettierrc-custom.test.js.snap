// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The prettierrc custom file should not change It should response with custom configuration 1`] = `
Object {
  "singleQuote": true,
}
`;
